front-end/src/renderer/pages/TransactionDetails/TransactionDetails.vue

```diff
@@ -528,6 +528,7 @@ const sectionHeadingClass = 'd-flex justify-content-between align-items-center';
 const detailItemLabelClass = 'text-micro text-semi-bold text-dark-blue';
 const detailItemValueClass = 'text-small mt-1';
 const commonColClass = 'col-6 col-lg-5 col-xl-4 col-xxl-3 overflow-hidden py-3';
+const descriptionColClass = 'col-12 col-lg-12 col-xl-12 col-xxl-12\'';
```

Comment:
There is no need for multiple col-12's
Bootstrap uses the min-width media queries so that you can leave only `col-12` and it will apply to any larger screens

Reply:
Then since it is only a single class, you don't need a variable for that